specification/containerservices/resource-manager/Microsoft.ContainerService/2017-07-01/containerService.json

```diff
@@ -0,0 +1,842 @@
+              "description": "Properties of the container service.",
+              "$ref": "#/definitions/ContainerServiceProperties",
+              "x-ms-client-flatten": true
+            }
+          }
+        }
+      ],
+      "description": "Container service."
+    },
+    "ContainerServiceListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerService"
+          },
+          "description": "The list of container services."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of container service results.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Container Services operation."
+    },
+    "ContainerServiceProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current deployment or provisioning state, which only appears in the response."
+        },
+        "orchestratorProfile": {
+          "$ref": "#/definitions/ContainerServiceOrchestratorProfile",
+          "description": "Profile for the container service orchestrator."
+        },
+        "customProfile": {
+          "$ref": "#/definitions/ContainerServiceCustomProfile",
+          "description": "Properties to configure a custom container service cluster."
+        },
+        "servicePrincipalProfile": {
+          "$ref": "#/definitions/ContainerServiceServicePrincipalProfile",
+          "description": "Information about a service principal identity for the cluster to use for manipulating Azure APIs. Exact one of secret or keyVaultSecretRef need to be specified."
+        },
+        "masterProfile": {
+          "$ref": "#/definitions/ContainerServiceMasterProfile",
+          "description": "Profile for the container service master."
+        },
+        "agentPoolProfiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerServiceAgentPoolProfile"
+          },
+          "description": "Properties of the agent pool."
+        },
+        "windowsProfile": {
+          "$ref": "#/definitions/ContainerServiceWindowsProfile",
+          "description": "Profile for Windows VMs in the container service cluster."
+        },
+        "linuxProfile": {
+          "$ref": "#/definitions/ContainerServiceLinuxProfile",
+          "description": "Profile for Linux VMs in the container service cluster."
+        },
+        "diagnosticsProfile": {
+          "$ref": "#/definitions/ContainerServiceDiagnosticsProfile",
+          "description": "Profile for diagnostics in the container service cluster."
+        }
+      },
+      "required": [
+        "orchestratorProfile",
+        "masterProfile",
+        "linuxProfile"
+      ],
+      "description": "Properties of the container service."
+    },
```

Comment:
@mboersma the semantic validation is failing because it cannot parse the JSON.
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/313171368 

I think the issue is an extra `,` here. Can you have a look please ?

Reply:
Thanks for noticing that. Fixed now.